templates/taxi/driver_list.html

```diff
@@ -2,6 +2,9 @@
 
 {% block content %}
     <h1>Driver List
+       <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
+      Create driver
+    </a>
     </h1>
```

Comment:
Code Style Issue: Consider adding a `Cancel` button that leads to the previous page the user was on. This will improve the user experience by providing an easy way to navigate back if they decide not to proceed with the current action.